Create DIVD-2026-00004.md (#1036)
DIVD-NL/web-csirt: _cases/2026/DIVD-2026-00004.md

---
layout: case
# Title and excerpt will be used on /cases and the RSS feed so make sure they reflect the case well

> title: Victim Notification stealerlog

Reviewer: ```suggestion
title: Victim Notification XenoRAT and PureLogs Stealer
```

> excerpt: "DIVD is notifying victims of the various infostealer malware strains from information.  If you received a notification, please read the appropriate instructions carefully."

Reviewer: ```suggestion
excerpt: "DIVD is notifying victims of XenoRAT and PureLogs Stealer from information.  If you received a notification, please read the appropriate instructions carefully."
```

author: Hans Meuris
lead: Frank Breedijk
researchers:
- Frank Breedijk
# You can use free text here as well. E.g. to indicate that some vulnerabilities don't have CVEs assigned (yet).
cves:
- n/a
product:
- n/a
#versions:
#- All versions
#- Again one per line
recommendation: If you received a notification from us, it means that you, members of your organization, or your customers likely have had their passwords stolen or their systems infected by the infostealer. Detailed recommendations can be found below.
#patch_status: Fully patched
#workaround: n/a
status : Open
start: 2026-04-16
end:
timeline:
- start: 2026-04-16

>   event: "Case file is public and first data avaiable"

Reviewer: ```suggestion
- start: 2026-02-14
  event: "First data is extracted from the C&C server"
- start: 2026-04-16
  event: "Case file is public and first data avaiable"
- start: 2026-05-01
  event: "Last data is harvested from the C&C server
- start: 2026-05-02
  event: "DIVD sends out the first notifications"
```

Author: case start date aangepast zodaat die overeenstemt met timeline start

ips: n/a
creds: 35000
# ips is used for statistics after the case is closed. If it is not applicable, you can set IPs to n/a (e.g. stolen credentials)
# This field becomes mandatory when the case status is set to 'Closed'
---
## Summary

> DIVD is doing their best to notify the victims in the various datasets.

Reviewer: ```suggestion

Through undisclosed channels DIVD has received data from a C&C server that is supporting either XenoRAT or PuraLogs Stealer infostealer malware, or both.

With this case we are notifying victims that we could identify from this set. Additionally we are publishing our usual apex lists to allow system administrators and system owners to assess if they have impacted users.
```

### Datasets involved

> So far we have received the following datasets:

Reviewer: ```suggestion
```

> ### The datasets contain

Reviewer: ```suggestion
### The datasets contains
```

> The datasets contain stolen user data, including usernames, redacted passwords, the URL and timestamp of the last observed use, and, where available, the victim’s IP address, device (computer) name, and local user account.

Reviewer: ```suggestion
The datasets contain stolen user data and has the following fields:
* User id (typically an email address)
* url - The usrl the passwor is for
* password - the (masked) password
* The source of the password - Typicallya browser program and the profile therein.
* 

We have enriched the data set with a window in which the data was stolen.
```

> ### Possible impact if no action is taken

Reviewer: ```suggestion
### Possible impact 
```

> The data on its own might appear harmless or insignificant, but when combined with reused passwords, lack of MFA (Multi-Factor Authentication), or internal host and user context, it makes account takeovers, fraud, and network intrusion possible. Taking action helps protect your accounts and personal information from being misused. 

Reviewer: ```suggestion
Consider that this data is compromised and in the hands of criminals. And that the account is potentially compromised.
```

## What you can do

> 

Reviewer: If you are the end user, here are a few steps you can take:

* If you recognize the password, change it immediately to prevent future abuse.
* If you’re unable to change your password because you are locked out of your account, contact your administrator and ask for a password change as soon as possible.
* Have you used this email address and password elsewhere? Change it immediately. Criminals often try the same username/password combinations on many different services and accounts.
* Never use this combination of email address and password anywhere else again. This email address and password is now noted in databases and traded amongst criminals, which makes them very easy to guess or crack.
* There might still be malware on your computer. Perform a virus scan to check if there is malware and take actions if there is.
* Enable dual or multi factor authentication (MFA). This stops an attack from criminals if they know your password. [implement two-step verification](https://ssd.eff.org/module/how-enable-two-factor-authentication) in as many places as possible.
* If this email address is a business email address, you are recommended to reach out to the security department of that organization.

### I / my users are on the list now what?

Here are the steps you can take depending on your situation:

> 

Reviewer: * Determine the severity of this situation based on the number of credentials we emailed you about.
* Investigate the accounts of these users for suspicious activities like large volumes of sent messages, or unusual log in times, or unusual mail rules.
* Consider scanning the devices of these users with malware detection tools.
* Dual factor authentication can stop an attacker that uses a compromised username and password combination from abusing the account [implement two-step verification](https://ssd.eff.org/module/how-enable-two-factor-authentication) in as many places as possible.
* Consider reporting this event to the appropriate authorities, e.g. law enforcement, privacy and other regulators.

## What we are doing

> As we receive more information, we will analyse them and publish more sets. Make sure you keep an eye on this space and our [CSIRT blog](/blog) or [RSS feed](https://csirt.divd.nl/feed.xml) 

Reviewer: The C&C server is now down. Apart for sending the notifications and answering your questions there is not much more for us to do.

## Frequently asked questions

### General

---

**Q: Is this a scam?**

A: It’s great that you’re skeptical. However, this is legit and definitely not a scam. 

---

**Q: Are you going to go after the criminals who stole my information?**

A: No, we are not. That is a matter for law enforcement. As per [article 9 of our code of conduct](https://www.divd.nl/code): `We analyze online threats, not threat actors. We are researchers and don’t serve the needs of governments or law enforcement.`

---

**Q: if you “don’t serve the needs of governments or law enforcement”, why are you cooperating with the Dutch National Police on this case?**

> A: Acting on this data set is directly in line with article 3 of our [code of conduct](https://www.divd.nl/code): `Analyze databases with leaked credentials and report to the organizations or people who are compromised to take appropriate measures.`

Reviewer: ```suggestion
```

We analyze every database we receive, including those from law enforcement. However, we do this independently, without any obligation or intention to share any specific information in return.

> 

Reviewer: ```suggestion
```

### Technical

---

**Q:  Do you know from which criminal operation my data was obtained?**

> A: In some cases we know the name of the infostealer malware, but further details were not shared with us.

Reviewer: ```suggestion
**Q:  Do you know when my data was stolen?**

A: Not really, but we know the oldest and newest timestamp iof other records in the C&C server.
```
